postgres-appliance/Dockerfile

```diff
@@ -44,12 +44,12 @@ RUN curl -L https://github.com/coreos/etcd/releases/download/v${ETCDVERSION}/etc
 
 ENV PATH $PATH:/usr/lib/postgresql-${PGVERSION}/bin
 
+# Copy the snakeoil certificates for usage as dummy certificates
+RUN cp /etc/ssl/private/ssl-cert-snakeoil.key /home/postgres/dummy.key && cp /etc/ssl/certs/ssl-cert-snakeoil.pem /home/postgres/dummy.crt
```

Comment:
You are right. In fact, I think PostgreSQL checks that the certificates are owned by postgres user and not accessible by anyone else, in the documentation there is a requirement to do chmod og-rwx server.key. So copying would be the only option.